plugin/path_impersonated_account_secrets_test.go

```diff
@@ -46,6 +47,22 @@ func testGetImpersonatedAccessToken(t *testing.T, roleName string) {
 		t.Fatalf("token email %q does not match service account email %q", info.Email, sa.Email)
 	}
 
+	// If the organizational policy which allows longer credential lifetime extensions to be used then
+	// verify that the TTL for this token is beyond the normal default
+	constraints, err := td.OrgAdmin.Organizations.Constraints.List(fmt.Sprintf("projects/%s", td.Project)).Do()
+	if err != nil {
+		t.Fatalf("error listing organizational constraints %q", err)
+	}
+	for _, constraint := range constraints.Constraints {
+		if strings.Contains(constraint.Name, "constraints/iam.allowServiceAccountCredentialLifetimeExtension") {
+			if constraint.ConstraintDefault == "ALLOW" {
+				if info.ExpiresIn < 4*60*60 {
+					t.Fatalf("expected a longer TTL on token but got: %d seconds", info.ExpiresIn)
+				}
+			}
+		}
+	}
```

Comment:
This might not be the correct code because it looks like Organization Policies can be confined to a particular service account as well, but there isn't a whole lot of documentation on how that is done in this case.  But it says:

> This list constraint defines the set of service accounts that can be granted OAuth 2.0 access tokens with a lifetime of up to 12 hours. By default, the maximum lifetime for these access tokens is 1 hour. The allowed/denied list of service accounts must specify one or more service account email addresses.